client/components/Categories/component.js

```diff
@@ -47,6 +45,22 @@ export default class CategoriesComponent extends Component {
     e.preventDefault();
 }
 
+  handleDelete = (e) => {
+    if(this.props.chosenCategoryId<0){
+      alert('You need to choose the category to delete!');
```

Comment:
I think that we should find some other way to show errors, because this alerts are not so nice xD But keep it for now.